Dockerfile

```diff
@@ -34,7 +36,9 @@ RUN chmod +x /tini
 # TODO: investigate
 RUN ln -s /bin/tar/ /bin/gtar
 
-RUN pip install civis-jupyter-notebook==${CIVIS_JUPYTER_NOTEBOOK_VERSION} && \
+RUN pip3 install wheel && \
+    pip3 install civis-jupyter-notebook==${CIVIS_JUPYTER_NOTEBOOK_VERSION} \
+      cbor2==4.1.2 && \
```

Comment:
Do you not need an `&& pip3 install` here?

Reply:
No, this is an additional package for `pip3 install` not a separate command (no `&&` at the EOL)